docs/concepts/work-pools.md

```diff
@@ -262,3 +269,95 @@ Together work queue priority and concurrency enable precise control over work. F
+| `--pool`, `-p` | The work pool the started worker should poll. |               
+| `--type`, `-t` | The type of worker to start. If not provided, the worker type will be inferred from the work pool. |   
+| <span class="no-wrap">`--prefetch-seconds`</span> | The amount of time before a flow run's scheduled start time to begin submission. Default is the value of `PREFECT_WORKER_PREFETCH_SECONDS`. |
+| `--run-once` | Only run worker polling once. By default, the worker runs forever. |
+| `--limit`, `-l` | The maximum number of flow runs to start simultaneously. |
+
+You must start a worker within an environment that can access or create the infrastructure needed to execute flow runs. The worker will deploy flow runs to the infrastructure corresponding to the worker type. For example, if you start a worker with type `kubernetes`, the worker will deploy flow runs to a Kubernetes cluster.
+
+!!! tip "Prefect must be installed in execution environments"
+    Prefect must be installed in any environment where you intend to run the worker or execute a flow run.
+
+!!! tip "`PREFECT_API_URL` setting for workers"
+    `PREFECT_API_URL` must be set for the environment in which your worker is running. 
+
+### Starting a Worker
+
+Use the `prefect worker start` CLI command to start a worker. You must pass at least the work pool name. If the work pool does not exist, it will be created if the `--type` flag is used.
+
+<div class="terminal">
+```bash
+$ prefect worker start -p [work pool name]
+```
+</div>
+
+For example:
+
+<div class="terminal">
+```bash
+prefect worker start -p "my-pool"
+Discovered worker type 'process' for work pool 'my-pool'.
+
+Worker 'ProcessWorker 65716280-96f8-420b-9300-7e94417f2673' started!
+```
+</div>
+
+In this case, Prefect automatically discovered the worker type from the work pool.
+
+To create a work pool and start a worker in one command, use the `--type` flag:
+
+<div class="terminal">
+```bash
+prefect worker start -p "my-pool" --type "process"
+
+Worker 'ProcessWorker d24f3768-62a9-4141-9480-a056b9539a25' started!
+06:57:53.289 | INFO    | prefect.worker.process.processworker d24f3768-62a9-4141-9480-a056b9539a25 - Worker pool 'my-pool' created.
+```
+</div>
+
+In addition, workers can limit the number of flow runs they will start simultaneously with the `--limit` flag. 
+
+For example, to limit a worker to five concurrent flow runs:
+
+<div class="terminal">
+```bash
+prefect worker start --pool "my-pool" --limit 5
+```
+</div>
+
+### Deployment Auto-Registration
+
+Workers can automatically apply discovered deployments. This behavior is useful for applying many deployments at once.
+
+To submit a deployment to a worker, move the deployment manifest and accompanying flow code to the worker's flows directory as defined by `PREFECT_WORKER_WORKFLOW_STORAGE_PATH` (defaults to `$PREFECT_HOME/workflows`). The worker will automatically discover and apply deployments when started. 
```

Comment:
Is this still WORKFLOW and not FLOW? I thought Chris +1'd that change in a previous PR